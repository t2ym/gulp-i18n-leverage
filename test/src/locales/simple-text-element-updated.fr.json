{
  "meta": {
    "todo": [
      {
        "op": "replace",
        "path": "/second-level-div:div_1",
        "value": "great grandchild text within div without id"
      },
      {
        "op": "add",
        "path": "/manual-object/name2",
        "value": "name 2"
      },
      {
        "op": "add",
        "path": "/manual-object/name3",
        "value": "name 3"
      },
      {
        "op": "add",
        "path": "/extra-array/4",
        "value": "array 4"
      },
      {
        "op": "add",
        "path": "/paragraph/1",
        "value": "id"
      }
    ]
  },
  "model": {
    "extra-model-object": {
      "attr1": {
        "extra-model-1": "fr extra model 1",
        "extra-model-2": "fr extra model 2",
        "extra-model-inner-array": [
          "fr extra model inner array 1",
          "fr extra model inner array 2"
        ]
      },
      "attr2": {
        "extra-attr-1": "fr extra attr 1",
        "extra-attr-2": "fr extra attr 2"
      }
    }
  },
  "text": "fr  outermost text at the beginning ",
  "h1_3": "fr outermost header 1",
  "text_4": "fr  outermost text in the middle ",
  "span_5": "fr simple text without id",
  "span_6": "fr simple text without id 2",
  "label-1": "fr simple text with id",
  "label-2": "fr simple text with id 2",
  "div_9:span": "fr simple text within div",
  "div_9:span_1": "fr simple text within div 2",
  "div_9:div_2:div": "fr great grandchild text within div",
  "div_10:text": "fr  simple text as the first element in div ",
  "div_10:span_1": "fr simple text within div",
  "div_10:text_2": "fr  simple text in the middle of div ",
  "div_10:span_3": "fr simple text within div 2",
  "div_10:div_4:div": "fr great grandchild text within div",
  "div_10:text_5": "fr  simple text at the last element in div ",
  "toplevel-div:span": "fr simple text within div",
  "toplevel-div:span_1": "fr simple text within div 2",
  "third-level-div": "fr great grandchild text within div",
  "second-level-div:div_1": "fr great grandchild text within div without id",
  "div_12:ul:li": "fr line item without id 1",
  "div_12:ul:li_1": "fr line item without id 2",
  "div_12:ul:li_2": "fr line item without id 3",
  "line-items:li": "fr line item with id 1",
  "line-items:li_1": "fr line item with id 2",
  "line-items:li_2": "fr line item with id 3",
  "p_13": [
    "fr A paragraph with {1} is converted to {2}.",
    "fr parameters",
    "fr <i18n-format>",
    "fr parameter 2",
    "fr parameter 3",
    "fr parameter 4",
    "fr parameter 5"
  ],
  "paragraph": [
    "fr A paragraph with {1} is converted to {2}.",
    "id",
    "fr <i18n-format>"
  ],
  "manual-object": {
    "name1": "fr name 1",
    "name2": "name 2",
    "name3": "name 3"
  },
  "extra-array": [
    "array 1",
    "fr array 2",
    {
      "extra-array-object-1": "fr object 1",
      "extra-array-object-2": "fr object 2"
    },
    "array 4"
  ],
  "text_15": "fr  outermost text at the end "
}
