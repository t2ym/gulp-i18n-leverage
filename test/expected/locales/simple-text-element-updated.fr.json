{
  "meta": {
    "todo": [
      {
        "op": "replace",
        "path": "/renamed-second-level-div:div_1",
        "value": "great grandchild text within div without id"
      },
      {
        "op": "add",
        "path": "/renamed-manual-object/name2",
        "value": "name 2"
      },
      {
        "op": "add",
        "path": "/renamed-manual-object/name3",
        "value": "name 3"
      },
      {
        "op": "add",
        "path": "/renamed-paragraph/1",
        "value": "id"
      },
      {
        "op": "add",
        "path": "/added-1",
        "value": "added text 1"
      },
      {
        "op": "add",
        "path": "/added-2",
        "value": "added text 2"
      },
      {
        "op": "add",
        "path": "/added-3",
        "value": "added text 3"
      },
      {
        "op": "replace",
        "path": "/span_5",
        "value": "updated simple text without id"
      },
      {
        "op": "replace",
        "path": "/span_6",
        "value": "updated simple text without id 2"
      },
      {
        "op": "replace",
        "path": "/label-1",
        "value": "updated simple text with id"
      },
      {
        "op": "replace",
        "path": "/label-2",
        "value": "updated simple text with id 2"
      },
      {
        "op": "replace",
        "path": "/div_9:span",
        "value": "updated simple text within div"
      },
      {
        "op": "replace",
        "path": "/div_9:span_1",
        "value": "updated simple text within div 2"
      },
      {
        "op": "replace",
        "path": "/div_10:text",
        "value": " updated simple text as the first element in div "
      },
      {
        "op": "replace",
        "path": "/div_10:span_1",
        "value": "updated simple text within div"
      },
      {
        "op": "replace",
        "path": "/div_10:text_2",
        "value": " updated simple text in the middle of div "
      },
      {
        "op": "replace",
        "path": "/div_10:span_3",
        "value": "updated simple text within div 2"
      },
      {
        "op": "replace",
        "path": "/div_10:text_5",
        "value": " updated simple text at the last element in div "
      },
      {
        "op": "replace",
        "path": "/toplevel-div:span",
        "value": "updated simple text within div"
      },
      {
        "op": "replace",
        "path": "/toplevel-div:span_1",
        "value": "updated simple text within div 2"
      },
      {
        "op": "replace",
        "path": "/p_13/0",
        "value": "An updated paragraph with {1} is converted to {2}."
      },
      {
        "op": "replace",
        "path": "/p_13/1",
        "value": "updated parameters"
      }
    ]
  },
  "model": {},
  "text": "fr  outermost text at the beginning ",
  "h1_3": "fr outermost header 1",
  "text_4": "fr  outermost text in the middle ",
  "span_5": "updated simple text without id",
  "span_6": "updated simple text without id 2",
  "label-1": "updated simple text with id",
  "label-2": "updated simple text with id 2",
  "div_9:span": "updated simple text within div",
  "div_9:span_1": "updated simple text within div 2",
  "div_9:div_2:div": "fr great grandchild text within div",
  "div_10:text": " updated simple text as the first element in div ",
  "div_10:span_1": "updated simple text within div",
  "div_10:text_2": " updated simple text in the middle of div ",
  "div_10:span_3": "updated simple text within div 2",
  "div_10:div_4:div": "fr great grandchild text within div",
  "div_10:text_5": " updated simple text at the last element in div ",
  "toplevel-div:span": "updated simple text within div",
  "toplevel-div:span_1": "updated simple text within div 2",
  "third-level-div": "fr great grandchild text within div",
  "renamed-second-level-div:div_1": "fr great grandchild text within div without id",
  "renamed-line-items:li": "fr line item with id 1",
  "renamed-line-items:li_1": "fr line item with id 2",
  "renamed-line-items:li_2": "fr line item with id 3",
  "p_13": [
    "An updated paragraph with {1} is converted to {2}.",
    "updated parameters",
    "fr <i18n-format>"
  ],
  "renamed-paragraph": [
    "fr A paragraph with {1} is converted to {2}.",
    "id",
    "fr <i18n-format>"
  ],
  "renamed-manual-object": {
    "name1": "fr name 1",
    "name2": "name 2",
    "name3": "name 3"
  },
  "text_15": "fr  outermost text at the end ",
  "added-1": "added text 1",
  "added-2": "added text 2",
  "added-3": "added text 3"
}